internal/services/controller/controller_test.go

```diff
@@ -63,10 +67,22 @@ func TestMain(m *testing.M) {
 func TestController_HappyPath(t *testing.T) {
 	tests := map[string]struct {
 		objectCount int
+		err         error
 	}{
 		"happy path": {
 			objectCount: 14,
 		},
+		"err when fetching api resources - multiple errors": {
```

Comment:
suggestion: the test name could be more explicit about given condition and expected result.

just an example
```suggestion
		"when fetching api resources produces multiple errors should ...": {
```